app/Models/Asset.php

```diff
@@ -96,6 +83,14 @@ public function declinedCheckout(User $declinedBy, $signature)
         'rtd_company_id' => 'integer',
         'supplier_id'    => 'integer',
         'byod'           => 'boolean',
+        'created_at'     => 'datetime',
+        'updated_at'   => 'datetime',
+        'deleted_at'  => 'datetime',
+        'purchase_date' => 'datetime',
+        'last_checkout' => 'datetime',
+        'expected_checkin'  => 'datetime',
+        'last_audit_date'  => 'datetime',
+        'next_audit_date' => 'datetime',
```

Comment:
Looks like these were already in the `$casts` array above and don't  need to be added here (we were doubling up I guess).

I think these lines can be removed since our original `$dates` array is incorrect and `purchase_date`, `expected_checkin`, and `next_audit_date` are actually dates.

Reply:
good catch, removed.